docs/storage/upload/README.md

```diff
@@ -0,0 +1,20 @@
+## upload sample
+
+This sample is a snippet of code with which you can create a CLI
+that allows you to upload a file to your Google Cloud Storage bucket.
+
+### Installing it
+```shell
+go get github.com/GoogleCloudPlatform/golang-samples/docs/storage/upload
```

Comment:
That's a great name, thanks! Sure I'll see about moving it. However, while waiting for @broady's response, I'll just do the rename, how about that?